domain-ee/ee-ep-merge-app/src/python_src/schema/merge_job.py

```diff
@@ -38,16 +37,20 @@ def __str__(self):
 
 
 class MergeJob(BaseModel):
-    _init_time: ClassVar[datetime] = datetime.now()
 
     job_id: UUID
     pending_claim_id: conint(strict=True)
     ep400_claim_id: conint(strict=True)
     state: JobState = JobState.PENDING
     error_state: JobState | None = None
     messages: list[Any] | None = None
-    created_at: datetime = _init_time
-    updated_at: datetime = _init_time
+    created_at: datetime = Field(default_factory=datetime.now)
+    updated_at: datetime = None
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        if "updated_at" not in kwargs or not self.updated_at:
```

Comment:
I think you probably only need one of these? Depending on whether you want to allow calling with `updated_at=None` or to enforce non-null.